### PR TITLE
main < fixed install script for macos

### PR DESCRIPTION
**Pull request by yacosta738**
fixed install script for macOS

bitmoji

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Organizer is a Rust script to organize the Download directory in diferent direct
 1. Install rustup.
 2. Download this repository.
 ```console
-git clone https://github.com/ghaerdi/file-organizer.git
+git clone https://github.com/yacosta738/file-organizer.git
```
Comment (ghaerdi) [inline README.md:13]: Revert this

**Review by ghaerdi — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please check the request changes.

File: README.md (hunk 1 above)
Comment (yacosta738) [reply README.md]: the repo URL reverted 

bitmoji

**Review by ghaerdi — APPROVED** [review]
**✅ LGTM**

**Pull request merged**